Gruntfile.js

```diff
@@ -30,6 +30,12 @@ module.exports = function (grunt) {
             },
             serve: {
                 command: 'source venv/bin/activate && python runserver.py'
+            },
+            test: {
+                command: 'ls ./assets/test/*.html | xargs -I % ./node_modules/.bin/mocha-headless-chrome -f %'
```

Comment:
what is the difference between 'test' and 'ci' here? is one your local development machine?

Reply:
The `test` target should work on any local developers machine, although obviously I've only been able to test that myself on my own machine. The `ci` target will only work in the CI environment (If we update the formula to install `chromium-browser`).

Reply:
As I mentioned above though, I don't really _like_ this solution much overall. Having 2 different behaviours for running the tests locally and then running them in CI is complexity that I don't really want to add and it's not the sort of 'canned' solution that wanted to come up with.
Everything is pointing to the fact that the tests just need re-writing, so I'm trying to persuade myself that this might be a resonable stop gap solution for now.

Reply:
elife-dashboard could do with some work in general:

* incorporate article-scheduler ([separate app](https://github.com/elifesciences/elife-article-scheduler), no end of headaches)
* move to Django app ([aborted attempt here](https://github.com/elifesciences/elife-dashboard/tree/develop/dashboard_2))
* replace BASIC login with something a touch more secure
* validation failure responses from lax when displayed are almost gibberish to read

And it was obvious from the beginning the frontend was overengineered. Now it can't be simply maintained when there are security issues either.

If we're not going to invest time in maintaining this app then we must settle for hacks ('stop-gap solutions') like this. Hacks are pure technical debt and our future selves will be saddled with it.

I'll open a ticket to get the test environments stitched up with chromium.